### PR TITLE
Rework, added docs.

### PR DESCRIPTION
**Pull request by ypid**
@drybjed Want to make a release first?

### CODE REVIEW

**Comment by drybjed** [comment]
@ypid I suppose it's a good idea, but right now I don't have access to my work computer, so this would need to wait a bit.

**Comment by ypid** [comment]
Sure. No hurry. But note that I will base new roles on this PR.

File: templates/etc/services.d/local_service.j2 [hunk 1]
```diff
@@ -1,22 +1,13 @@
...
+{% elif item.name|d() and item.port|d() %}
+{%   set etc_services__tpl_protocols = (item.protocols|d(item.protocol)|d([ 'tcp', 'udp' ])) | list %}
+{%   for transport_layer_protocol in etc_services__tpl_protocols %}
+{{     "%-15s %s/%-26s%s" | format(
+          item.name,
+          item.port,
+          transport_layer_protocol,
+          (" # " + item.comment) if (item.comment|d()) else "")
+}}
+{%   endfor %}
```
Comment (drybjed) [inline templates/etc/services.d/local_service.j2]: That's some serious compression. :-) Very nice.

**Comment by drybjed** [comment]
@ypid Excellent work, as always. :-) I'll merge this as soon as possible.

**Comment by ypid** [comment]
Thanks very much.

**Pull request merged**